pkg/fixtures/test/panic_on_no_return_value_test.go

```diff
@@ -0,0 +1,29 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	mocks "github.com/vektra/mockery/v2/mocks/github.com/vektra/mockery/v2/pkg/fixtures"
+)
+
+func TestPanicOnNoReturnValue(t *testing.T) {
+	m := mocks.NewPanicOnNoReturnValue(t)
+	m.EXPECT().DoSomething()
+
+	var panicOccurred bool
+	defer func() {
+		assert.True(t, panicOccurred)
+	}()
+	defer func() {
+		panicOccurred = true
+		if r := recover(); r != nil {
+			require.NotNil(t, r)
```

Comment:
Actually one more thing about this. Don't use the if statement, it's possible in this test that we're silently not asserting the contents of `r` (in the case it's nil.

```go
r := recover()
require.NotNil(t, r)
assert....
```

Reply:
Great, thanks!